src/js/view/captionsrenderer.js

```diff
@@ -200,7 +200,7 @@ const CaptionsRenderer = function (viewModel) {
 
         let fontSize;
         if (_model.get('fullscreen') && OS.iOS) {
-            fontSize = 'inherit';
+            fontSize = null;
```

Comment:
When going fullscreen we want to clear the custom rules, not set them to inherit. This only appeared to work as expected because the value 'inherit' was made invalid by appending 'px' below.

Reply:
🍀